conda/recipes/cudf/meta.yaml

```diff
@@ -17,10 +17,22 @@ build:
   number: {{ GIT_DESCRIBE_NUMBER }}
   string: cuda_{{ cuda_major }}_py{{ py_version }}_{{ GIT_DESCRIBE_HASH }}_{{ GIT_DESCRIBE_NUMBER }}
   script_env:
-    - VERSION_SUFFIX
     - PARALLEL_LEVEL
-  # libcudf's run_exports pinning is looser than we would like
+    - VERSION_SUFFIX
+    - CMAKE_GENERATOR
+    - CMAKE_C_COMPILER_LAUNCHER
+    - CMAKE_CXX_COMPILER_LAUNCHER
+    - CMAKE_CUDA_COMPILER_LAUNCHER
+    # TODO: remove "gha" suffix before PR is merged
```

Comment:
```suggestion
```

Reply:
whoops. :facepalm:. i'll go back and remove these after the current builds are complete and the cache is re-populated.